src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
```

Comment:
you've set `box-sizing: border-box;` at `*` selector
So no needs to set it again. Remove all over the card